### PR TITLE
[INLONG-9337][Manager] Support querying operation records

### LINKED ISSUE #9337: [Improve][Manager] Support querying operation records
### Description

Support querying operation records。

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
- Fixes #9337 

### Motivation

Support querying operation records.
1.For `operation_log` table has added `operation_target`, `inlong_stream_id`, `inlong_group_id` field.
2.supports querying operation log records based on `inlongGroupId`, `inlongStreamId`, and operation request time.
3.The return values are operation time, operation object, and whether the operation was successful.

Through this query operation record, it is possible to query which operations were performed in a certain `inlongGroup` during a certain time period, whether these operations were successful, and effectively record the parameters of information changes.

### Modifications

Support querying operation records.

### CODE REVIEW

**Review by vernedeng — APPROVED** [review]

File: inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml [hunk 1]
```diff
@@ -48,25 +51,52 @@
         from operation_log
         where id = #{id,jdbcType=INTEGER}
     </select>
+    <select id="selectByCondition" resultMap="BaseResultMap"
+            parameterType="org.apache.inlong.manager.pojo.operationLog.OperationLogRequest">
+        select
+        <include refid="Base_Column_List"/>
+        from operation_log
+        <where>
+            <if test="request.inlongGroupId != null and request.inlongGroupId != ''">
```
Comment (luchunliang) [inline inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:60]: groupId and streamId need fuzzy search.

File: inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml [hunk 2]
```diff
@@ -48,25 +51,52 @@
...
+                and inlong_group_id = #{request.inlongGroupId, jdbcType=VARCHAR}
+            </if>
+            <if test="request.inlongStreamId != null and request.inlongStreamId != ''">
```
Comment (luchunliang) [inline inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:63]: How to fuzzy search datanode or inlong cluster log?

Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:63]: Currently not considering nodes and clusters.

File: inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml [hunk 3]
```diff
@@ -48,25 +51,52 @@
...
+                and inlong_stream_id = #{request.inlongStreamId, jdbcType=VARCHAR}
+            </if>
+            <if test="request.operationType != null and request.operationType != ''">
+                and operation_type = #{request.operationType, jdbcType=VARCHAR}
+            </if>
+            <if test="request.operationTarget != null and request.operationTarget != ''">
+                and operation_target = #{request.operationTarget, jdbcType=VARCHAR}
+            </if>
+            <if test="request.startDate != null and request.startDate != '' and request.endDate != null and request.endDate != ''">
+                and request_time &gt;= #{request.startDate, jdbcType=VARCHAR}
```
Comment (luchunliang) [inline inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:73]: RequestTime type is timestamp.
Please confirm that there will be no performance issues here.

Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:73]: There won't be any problems.

File: inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/resources/mappers/OperationLogEntityMapper.xml:60]: Done.

File: inlong-manager/manager-web/sql/changes-1.10.0.sql [hunk 4]
```diff
@@ -36,4 +36,15 @@ VALUES ('audit_sort_mysql_binlog_input', 'MYSQL_BINLOG', 0, '29'),
...
+    ADD COLUMN  `inlong_group_id`  varchar(256) DEFAULT NULL COMMENT 'Inlong group id';
+
+ALTER TABLE `operation_log`
+    ADD COLUMN  `inlong_stream_id` varchar(256) DEFAULT NULL COMMENT 'Inlong stream id',
+
+ALTER TABLE `operation_log`
+    ADD COLUMN `operation_target` varchar(256) DEFAULT NULL COMMENT 'Operation target',
+
+CREATE INDEX operation_log_group_stream_index ON operation_log (`inlong_group_id`, `inlong_stream_id`);
+
```
Comment (luchunliang) [inline inlong-manager/manager-web/sql/changes-1.10.0.sql:49]: request_time field is high-frequency query conditions, but it is not in any key index.

Comment (fuweng11) [reply inlong-manager/manager-web/sql/changes-1.10.0.sql:49]: Done.

**Review by vernedeng — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**